client/styles/components/_sidebar.scss

```diff
@@ -95,6 +95,10 @@
 .file-item__content {
   display: flex;
   position: relative;
+  padding-top: #{12 / $base-font-size}rem;
+  padding-bottom: #{12 / $base-font-size}rem;
+  border-top: 1px solid #f4f4f4;
```

Comment:
could you pull this color from the theme map so it looks good in the light dark, and high contrast themes?